net.brinkervii.grapejuice.yml

```diff
@@ -0,0 +1,122 @@
+app-id: net.brinkervii.grapejuice 
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: grapejuice
+separate-locales: false
+
+finish-args:
+        - --share=network
+        - --socket=x11
+        - --share=ipc
+        - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
Yes

Reply:
then why `--socket=x11` and not `fallback-x11` ?

Reply:
I misread the documentation, fixing with a new commit

Reply:
Hello, it might be necessary to still use `--socket=x11`, as the provided wine build isn't capable of running under wayland.

Currently, the x11 fallback makes wayland users unable to run ROBLOX via wine. The Grapejuice GTK UI, on the other hand, renders and behaves properly on wayland.